examples/src/main/scala/sttp/tapir/examples/observability/OpenTelemetryMetricsExample.scala

```diff
@@ -0,0 +1,93 @@
+package sttp.tapir.examples.observability
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Route
+import com.typesafe.scalalogging.StrictLogging
+import io.circe.generic.auto._
+import io.opentelemetry.api.OpenTelemetry
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter
+import io.opentelemetry.sdk.OpenTelemetrySdk
+import io.opentelemetry.sdk.metrics.SdkMeterProvider
+import io.opentelemetry.sdk.metrics.`export`.PeriodicMetricReader
+import sttp.tapir._
+import sttp.tapir.generic.auto._
+import sttp.tapir.json.circe.jsonBody
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.akkahttp.{AkkaHttpServerInterpreter, AkkaHttpServerOptions}
+import sttp.tapir.server.metrics.opentelemetry.OpenTelemetryMetrics
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+/** This example uses a gRPC <a href="https://opentelemetry.io/docs/concepts/components/#exporters">exporter</a> to send metrics to a <a href="https://opentelemetry.io/docs/collector/">collector</a>, which by
+  * default is expected to be running on `localhost:4317`.
+  *
+  * You can run a collector locally using Docker with the following command:
```

Comment:
**suggestion:** Maybe we should also suggest the three `libraryDependencies` that you've added in build.sbt?

Reply:
Good point, added